.linkcheckerrc.ini

```diff
@@ -0,0 +1,9 @@
+[filtering]
+# we ignore _static files which include jupyter-book templating features
+# which would otherwise cause errors during link checks
+ignore=
+  ^html/_static
+[output]
+# ignore issues with github issues links showing 404's
```

Comment:
```suggestion
# ignore errors with github issues links incorrectly reading 404
```

To increase clarity, but maybe I missed the mark so please fix if necessary

Reply:
Thanks for this, I think it's an improvement!